modules/cluster-limitations-local-zone.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+[id="cluster-limitations-local-zone_{context}"]
+= Cluster limitations in AWS Local Zones
+
+Some limitations exist when you attempt to deploy a cluster with a default installation configuration in Amazon Web Services (AWS) Local Zones.
+
+[IMPORTANT]
+====
+The following list details limitations when deploying a cluster in AWS Local Zones:
+
+- The Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is `1300`. This causes the cluster-wide network MTU to change according to the network plugin that is used on the deployment.
+- Network resources such as Network Load Balancer (NLB), Classic Load Balancer, and Nat Gateways are not supported in AWS Local Zones.
+- The AWS Elastic Block Storage (EBS) `gp3` type volume is the default for node volumes and the default for the storage class set on AWS OpenShift clusters. This volume type is not globally available on Local Zone locations. By default, the nodes running in Local Zones are deployed with the `gp2` EBS volume. The `gp2-csi` `StorageClass` must be set when creating workloads on Local Zone nodes.
```

Comment:
Updated but went with `For an {product-title} cluster on AWS` because OpenShift on its own or without the preceding "Red Hat" tag might cause stylistic issues.